src/render/alt-tree-mui.tsx

```diff
@@ -357,7 +357,11 @@ function markNode(t: AltTrack | AltTrackOverlaidByDataInd, uid: string): AltNode
         mark = `The mark used is ${t.appearance.details.mark}.`;
     }
     if (t.alttype === 'ov-mark') {
-        mark = `The marks used are ${arrayToString([t.appearance.details.mark, ...t.appearance.details.markByTrack])}.`;
+        if (t.appearance.details.markByTrack) {
+            mark = `The marks used are ${arrayToString([t.appearance.details.mark, ...t.appearance.details.markByTrack])}.`;
```

Comment:
Just curious, what is the difference between `t.appearance.details.mark` and `t.appearance.details.markByTrack`? Is this a track with multiple child tracks overlaid, and you are collecting marks at both the parent and child levels?

Reply:
yes! :)